src/components/directory-content.js

```diff
@@ -99,7 +99,7 @@ const DirectoryContent = () => {
     const dispatch = useDispatch();
 
     const [childrenMetadata, setChildrenMetadata] = useState({});
-    const [isAllDataPresent, setIsAllDataPresent] = useState(false);
+    const isAllDataPresent = true;
```

Comment:
Since isAllDataPresent is always true (it seems it is never modified), maybe it is possible to completely remove it ?

Reply:
You are right. I am removing it.